commitizen/cz/customize/customize.py

```diff
@@ -79,3 +83,53 @@ def info(self) -> Optional[str]:
         elif info:
             return info
         return None
+
+
+class CustomizeCommitValidationCz(CustomizeCommitsCz):
```

Comment:
It's mostly so that I could test it in the unit tests, but also to give others an example to work with if they want to use the feature. I didn't see a better place to put it, if you can suggest one I'm happy to put it somewhere :)